public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
```

Comment:
if you like, you can get rid of the react favicon! If you wanna leave it there, please ignore me lol 😅